Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -79,15 +128,28 @@
     'vtPositives': 10,
     'vtPositiveUrlsForIP': 30
 }
-# The dictionary below does not represent DBot Scores correctly, and should not be used
-dbotscores = {
```

Comment:
Why did we remove this?

Reply:
it is not used anywhere, so I deleted it